/yam_l:
  ros__parameters:
    arm_can_id: can_yam_l
    arm_control_type: normal
    arm_pub_topic_name: arm_l_status
    arm_sub_topic_name: YAM_VR_L
    arm_control_period: 0.004

    # gain parameters
    arm_kp_1_3: 120.0   # kp for joints
    arm_kp_4_6: 30.0
    arm_kd_1_3: 2.0
    arm_kd_4_6: 1.0     # kd for joints
    gripper_position_gain: 30.0 # kp for gripper
    gripper_velocity_gain: 1.0  # kd for gripper
    go_home_kp : 25.0
    go_home_kd : 0.5


/yam_r:
  ros__parameters:
    arm_can_id: can_yam_r
    arm_control_type: normal
    arm_pub_topic_name: arm_r_status
    arm_sub_topic_name: YAM_VR_R
    arm_control_period: 0.004

    # gain parameters
    arm_kp_1_3: 120.0   # kp for joints
    arm_kp_4_6: 30.0
    arm_kd_1_3: 2.0
    arm_kd_4_6: 1.0     # kd for joints
    gripper_position_gain: 30.0 # kp for gripper
    gripper_velocity_gain: 1.0  # kd for gripper
    go_home_kp : 25.0
    go_home_kd : 0.5
